codewithher/CC-Lantern
# 22 Migrate Color Wheel Activity #22

**Why:**

Closes the following issues (all PRs must close an issue):

- closes #22 


**What's being changed**

Simplifies the rainbow() method in act3 examples to implement Lantern.h library, making the function easier to read. 


**Testing instructions**

The easiest way to test is to run the rainbow method in activity 3 which double as testing files. 

You can verify that the code works appropriately by visually inspecting the colors that result from the activity function calls.

- [x] The LED goes through the complete color spectrum (red to green to blue back to red)
- [x] The color change is gradual and smooth 
- [x] The rainbow lasts the specified "seconds" amount of time
- [x] Change the number of seconds and make sure the rainbow lasts appropriate amount of time 


**Check off the following**

- [x] I have assigned @totally-not-frito-lays as a reviewer
